pkgs/development/libraries/qt-6/modules/qtbase.nix

```diff
@@ -59,13 +61,16 @@ let
   qmakeCacheName = ".qmake.stash";
   debugSymbols = debug || developerBuild;
 
-  cacheBuildPhaseResult = true; # save the result of buildPhase in a separate derivation
-  # make easier to debug postInstall, postFixup
+  splitBuildInstall = true;
```

Comment:
TODO generalize this for `stdenv.mkDerivation`

useful for debugging large builds = buildPhase runs longer than 5 minutes

the option `splitBuildInstall` can be enabled to get a "development mode" for a package,
and as soon as the package is "ready for production",
we remove `splitBuildInstall` to run buildPhase and installPhase in one derivation